### PR TITLE
improve student questions view (#284)(#276)

### LINKED ISSUE #276: Dodać labelkę do widoku pytań studenckich ze strony wykładowcy

### LINKED ISSUE #284: Poprawa implementacji listy pytań od studentów
Nowe pytania są oznaczone (jak na discordzie kreską nad  nimi)
Pusta lista dalej jest na całą wysokość
Poprawa godzin żeby minuty miały zawsze 2 cyfry i żeby to było zaimplementowane na Date()

### PR DESCRIPTION
**Pull request by Petros9**
the `Pytania od studentów` bar is added, the new question are now marked by red color 
image

### CODE REVIEW

**Review by rozchlastywacz — APPROVED** [review]

File: src/lecturer/components/notifiableTab/NotifiableTab.tsx [hunk 1]
```diff
@@ -21,21 +22,22 @@ const StyledBadge = withStyles((theme: Theme) =>
     })
 )(Badge);
 export default function NotifiableTab(props: NotifiableTabProps) {
-
+    const nothing = () =>{}
     const [notifiableNumber, setNotifiableNumber] = useState(0);
     const label = props.label ?? "none";
     const routes = props.routes ?? "";
     const value = props.value ?? "";
+    const resetFunction = props.resetFunction?? nothing;
```
Comment (Derstilon) [inline src/lecturer/components/notifiableTab/NotifiableTab.tsx]:
```suggestion
    const [notifiableNumber, setNotifiableNumber] = useState(0);
    const label = props.label ?? "none";
    const routes = props.routes ?? "";
    const value = props.value ?? "";
    const resetFunction = props.resetFunction ?? (() =>{ });
```
This is much cleaner

File: src/lecturer/components/notifiableTab/NotifiableTab.tsx [hunk 2]
```diff
@@ -21,21 +22,22 @@ const StyledBadge = withStyles((theme: Theme) =>
     })
 )(Badge);
 export default function NotifiableTab(props: NotifiableTabProps) {
-
+    const nothing = () =>{}
     const [notifiableNumber, setNotifiableNumber] = useState(0);
     const label = props.label ?? "none";
     const routes = props.routes ?? "";
     const value = props.value ?? "";
```
Comment (Derstilon) [inline src/lecturer/components/notifiableTab/NotifiableTab.tsx]: You don't use `value` anywhere in this component

File: src/lecturer/components/studentsQuestionView/StudentsQuestionListView.tsx [hunk 3]
```diff
@@ -64,28 +84,47 @@ export function StudentsQuestionListView() {
...
+                            ):(
+                                <TextField className={classes.field} error variant="outlined"
+                                 fullWidth={true} 
+                                 multiline
+                                 label={studentQuestion.time.toLocaleTimeString("en-GB") + " | Anonimowy student"} 
+                                 defaultValue={studentQuestion.text} InputProps={{
+                                    className: classes.questionText,
+                                    readOnly: true,
+                                }}/>
+                            )}
```
Comment (Derstilon) [inline src/lecturer/components/studentsQuestionView/StudentsQuestionListView.tsx]:
```suggestion
                                <TextField className={classes.field} error={!studentQuestion.processed} variant="outlined"
                                 fullWidth={true} 
                                 multiline
                                 label={studentQuestion.time.toLocaleTimeString("en-GB") + " | Anonimowy student"} 
                                 defaultValue={studentQuestion.text} InputProps={{
                                    className: classes.questionText,
                                    readOnly: true,
                                }}/>
```
You could remove this duplicating code by moving "if" statement into the  "error" property

**Review by Derstilon — CHANGES_REQUESTED** [review]
I've added some code refactor suggestions and new issues for the bugs I've seen. #304 #305  Please add those changes, write a description for those tasks for future reference and let me know when you'll be done so that I can approve PR.

**Review by Derstilon — APPROVED** [review]

**Pull request merged**